### PR TITLE
Aggiunta funzione per il calcolo delle prestazioni in autorotazione

### PR DESCRIPTION
**Pull request by giulioguarino**

### CODE REVIEW

**Comment by giulioguarino** [comment]
We have two doubts about the function.

1) Solving  the first equation (in respect to theta_0) yields two solutions. Is there a way to choose the one with physical meaning? 
   We arbitrarily chose the second one because it seemed more reasonable.

 2) The code doesn't seem to work for values of advance ratio which are equal to or less than 0.1. We suspect this might be because of the theoretical limits of the procedure described in reference.

File: prestazioni_autorotazione.m [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+% |Output      : Numerical values of thrust coefficient, angle of                              |
+%                attack, angular velocity, freestream velocity, tip                            |
+%                velocity.                                                                     | 
+% |Note        : note addizionali                                                              |
+% ==============================================================================================
+
+function [T_C, H_C, alpha_deg, omega, V_inf ] = prestazioni_autorotazione(mu,X)
+
+%Supposing helicopter data:
+global  R...                          % [m], rotorblade radius
```
Comment (EttoreSaetta) [inline prestazioni_autorotazione.m]: Do not use global variables.

File: prestazioni_autorotazione.m [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+% |Reference   : Lezioni di Aerodinamica dell'Ala Rotante a.a. 2019/2020                       |
+%                Renato Tognaccini                                                             | 
+% |Input       : Advance ratio, descent ratio values (to be assigned).                         |
+% |Output      : Numerical values of thrust coefficient, angle of                              |
+%                attack, angular velocity, freestream velocity, tip                            |
+%                velocity.                                                                     | 
+% |Note        : note addizionali                                                              |
+% ==============================================================================================
+
+function [T_C, H_C, alpha_deg, omega, V_inf ] = prestazioni_autorotazione(mu,X)
```
Comment (EttoreSaetta) [inline prestazioni_autorotazione.m]: The name of the function should be in English.

**Review by EttoreSaetta — COMMENTED** [review]
Some comments here, you also need to upload a documentation for starting a review.
Moreover put the test case in an appropriate folder.
As regards your theoretical questions we can discuss about it during our meetings.
You can join our Tuesday meetings for these questions.

**Comment by giulioguarino** [comment]
This should be the definitive version of the function. We are waiting for a final review.

**Review by EttoreSaetta — APPROVED** [review]

**Pull request merged**